### PR TITLE
New new new

### PR DESCRIPTION
**Pull request by pietbrauer**
New Site!!

### CODE REVIEW

File: _posts/2015-03-15-git2go.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: "Git2Go"
+category: app
+icon: "/assets/Git2Go.svg"
+app_store_link: "http://git2go.com"
+---
+
+## Is it released yet? I am waiting for it.
+
+@tapwork, [Heap Inspector](https://github.com/tapwork/HeapInspector-for-iOS) and iOS Developer
```
Comment (pietbrauer) [inline _posts/2015-03-15-git2go.md:10]: @tapwork are you ok with your mention?
tapwork_quote

File: _posts/2015-01-06-airportquiz.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+category: app
+title: "AirportQuiz"
+icon: "/assets/AirportQuiz.svg"
+app_store_link: "https://itunes.apple.com/us/app/airport-quiz/id898891728?ls=1&mt=8"
+---
+
+## What can I say? I am ruling it.
+
+@daehn, #1 in all AirportQuiz leaderboards
```
Comment (pietbrauer) [inline _posts/2015-01-06-airportquiz.md:10]: @daehn are you ok with your mention?
daehn_quote

Comment (daehn) [inline _posts/2015-01-06-airportquiz.md:10]: Yes I am!

File: _posts/2015-03-15-git2go.md (hunk 1 above)
Comment (tapwork) [inline _posts/2015-03-15-git2go.md:10]: yeah cool
first I thought about app release.
how about "Is memory freed?"

Comment (pietbrauer) [inline _posts/2015-03-15-git2go.md:10]: I think the quote should reflect your excitement for Git2Go :wink:

**Pull request merged**